apps/mobile/app/utils/crashReporting.ts

```diff
@@ -23,46 +23,45 @@
  *  This is where you put your crash reporting service initialization code to call in `./app/app.tsx`
  */
 export const initCrashReporting = () => {
-  // Sentry.init({
-  //   dsn: "YOUR DSN HERE",
-  //   enableInExpoDevelopment: true,
-  //   debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
-  // })
-  // Bugsnag.start("YOUR API KEY")
+	Sentry.init({
+		dsn: process.env.EXPO_PUBLIC_SENTRY_DSN,
+		enableInExpoDevelopment: true,
+		debug: __DEV__, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
+	})
+	// Bugsnag.start("YOUR API KEY")
 }
-
 /**
  * Error classifications used to sort errors on error reporting services.
  */
 export enum ErrorType {
-  /**
-   * An error that would normally cause a red screen in dev
-   * and force the user to sign out and restart.
-   */
-  FATAL = "Fatal",
-  /**
-   * An error caught by try/catch where defined using Reactotron.tron.error.
-   */
-  HANDLED = "Handled",
+	/**
+	 * An error that would normally cause a red screen in dev
+	 * and force the user to sign out and restart.
+	 */
+	FATAL = "Fatal",
+	/**
+	 * An error caught by try/catch where defined using Reactotron.tron.error.
+	 */
+	HANDLED = "Handled",
 }
 
 /**
  * Manually report a handled error.
  */
 export const reportCrash = (error: any, type: ErrorType = ErrorType.FATAL) => {
-  if (__DEV__) {
-    // Log to console and Reactotron in development
-    const message = error.message || "Unknown"
-    console.error(error)
-    console.log(message, type)
-    console.tron.log(error)
-  } else {
-    // In production, utilize crash reporting service of choice below:
-    // RN
-    // Sentry.captureException(error)
-    // Expo
-    // Sentry.Native.captureException(error)
-    // crashlytics().recordError(error)
-    // Bugsnag.notify(error)
-  }
+	if (__DEV__) {
+		// Log to console and Reactotron in development
+		const message = error.message || "Unknown"
+		console.error(error)
+		console.log(message, type)
+		console.tron.log(error)
+	} else {
+		// In production, utilize crash reporting service of choice below:
+		// RN
+		// Sentry.captureException(error)
+		// Expo
+		Sentry.Native.captureException(error)
```

Comment:
Same, add here  check if `process.env.EXPO_PUBLIC_SENTRY_DSN` is defined and only call that method if yes